### PR TITLE
new config approach

### PR DESCRIPTION
**Pull request by alex311982**
Final work

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 server/www
+.idea
```
Comment (pattt) [inline .gitignore]: add empty line

File: bookshop/.env.example [hunk 2]
```diff
@@ -0,0 +1 @@
+API_URL=http://api.local:8081/api/
```
Comment (pattt) [inline bookshop/.env.example]: use url that compatible with docker container

Comment (pattt) [reply bookshop/.env.example]: add empty line

File: .gitignore (hunk 1 above)
Comment (alex311982) [reply .gitignore]: ok

File: bookshop/.env.example (hunk 2 above)
Comment (alex311982) [reply bookshop/.env.example]: ok

**Comment by pattt** [comment]
:shipit:

**Review by pattt — APPROVED** [review]

**Pull request merged**